kairon/shared/data/collection_processor.py

```
@@ -1,9 +1,231 @@
+        except DoesNotExist:
+            raise AppException("Collection Data with given id and collection_name not found!")
+
+        return collection_id
+
+    @staticmethod
+    def delete_collection_data( collection_id: str, bot: Text, user: Text):
+        try:
+            collection = CollectionData.objects(bot=bot, id=collection_id).get()
+            collection.delete(user=user)
+        except DoesNotExist:
+            raise AppException("Collection Data does not exists!")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)